hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/StreamWriteOperatorCoordinator.java

```diff
@@ -362,6 +362,10 @@ private void initClientIds(Configuration conf) {
 
   private void reset() {
     this.eventBuffer = new WriteMetadataEvent[this.parallelism];
+    if (!this.instant.equals(WriteMetadataEvent.BOOTSTRAP_INSTANT)) {
+      writeClient.getHeartbeatClient().stop(instant);
```

Comment:
This should check whether the failed writes clean policy is lazy.

Reply:
done. And move it to `initInstant` method.